### PR TITLE
German translation 1

### PR DESCRIPTION
**Pull request by MHumm**
Further German translations

### CODE REVIEW

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 1]
```diff
@@ -208,25 +209,22 @@ msgstr "Neu:"
 #  Programmer's name for it: RsHLEdPropDlg_lblColor
 #. JvFullColorCircleFrm..LabelColorSpace..Caption
 #: JvFullColorCircleForm.dfm:45
-#, fuzzy
 msgid "&Color Space:"
-msgstr "&Farbe:"
+msgstr "&Farbraum"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: The colon is dropped here.

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 2]
```diff
@@ -1153,7 +1150,7 @@ msgstr ""
 msgid ""
 "Tell the component\n"
 "which part you want of the string"
-msgstr ""
+msgstr "Hier der Komponente mitteilen welcher String-Teil gewünscht ist"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: Should be a comma after "mitteilen".

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 3]
```diff
@@ -1184,7 +1181,7 @@ msgstr "Alles nach dem Start-Tag"
 #. JvHTMLParserForm..GroupBox1..cbTakeText....Items.Strings
 #: JvParserForm.dfm:130
 msgid "The whole line if respecting the condition"
-msgstr ""
+msgstr "Die ganze Zeile wenn die Bedingung erfüllt ist"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: Should be a comma after "Zeile"

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 4]
```diff
@@ -1203,6 +1200,12 @@ msgid ""
 "2 if you want it in the second position\n"
 "...."
 msgstr ""
+"Hier die Position des Start Tags definieren\n"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: Either "Starttags" or "Start-Tags".

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 5]
```diff
@@ -1203,6 +1200,12 @@ msgid ""
...
+"\n"
+"1 wenn es egal ist\n"
+"0 wenn er nicht im string sein kann\n"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: Even if "string" is an internal type, here it should be uppercase as it's usesld as a name here.

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 6]
```diff
@@ -1517,7 +1518,7 @@ msgstr "&Einfügen"
 #. frmSLDMappingEditorDialog..EditorFrame..alCharMapEditor..aiEditClear..Caption
 #: JvSegmentedLEDDisplayMapperFrame.dfm:135 JvSLDMappingEditorDialog.dfm:251
 msgid "&Empty segments"
-msgstr ""
+msgstr "&Leere segmente"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: "Segmente" has to be uppercase.

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 7]
```diff
@@ -1571,7 +1572,7 @@ msgstr ""
 #. frmSLDMappingEditorDialog..lblDigitClassCaption..Caption
 #: JvSLDMappingEditorDialog.dfm:26
 msgid "Digit class:"
-msgstr ""
+msgstr "Ziffern Klasse:"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: One word: "Ziffernklasse"

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 8]
```diff
@@ -2234,6 +2233,8 @@ msgid ""
 "The Default Root Value \"%0:s\" has been replaced with \"%1:s\".\n"
 "Please change the value in the FileVersionInfo Project Properties."
 msgstr ""
+"Der Standard Wurzel Wert \"%0:s\" wurde durch \"%1:s\" ersetzt.\n"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: "Der standard Wurzelwert"

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 9]
```diff
@@ -2359,7 +2360,7 @@ msgstr "'%s' ist kein gültiger boolescher Wert"
 #. Programmer's name for it: RsENotABooleanValue
 #: JvResources.pas:170
 msgid "XML Node names cannot contain white space and the WhiteSpaceReplacement property is empty. Please set the WhiteSpaceReplacementProperty to a non empty value."
-msgstr ""
+msgstr "XML Konten Namen können keine Leerzeichen enthalten und die WhiteSpaceReplacement Eigenschaft ist leer. Bitte WhiteSpaceReplacement Eigenschaft auf einen nicht leeren Wert setzen"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: "Konten" should be "Knoten" (and it should be "XML-Knoten"), missing dot at the end.

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 10]
```diff
@@ -2369,12 +2370,12 @@ msgstr "Die WhiteSpaceReplacement-Property darf selbst keinen Whitespace enthalt
 #. Programmer's name for it: RsENotABooleanValue
 #: JvResources.pas:174
 msgid "XML Node names cannot contain invalid chars (\"%s\") and the InvalidCharReplacement property is empty. Please set the InvalidCharReplacement to a non empty value."
-msgstr ""
+msgstr "XML Knoten dürfen keine ungültigen zeichen enthalten (\"%s\") und die InvalidCharReplacement Eigenschaft ist leer. Bitte InvalidCharReplacement  Eigenschaft auf einen nicht leeren Wert setzen."
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: "XML-Knoten Namen" and "Zeichen"

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 11]
```diff
@@ -2369,12 +2370,12 @@ msgstr "Die WhiteSpaceReplacement-Property darf selbst keinen Whitespace enthalt
...
 
 #. Programmer's name for it: RsENotAllowedCharacterForProperty
 #: JvResources.pas:175
 msgid "\"%0:s\" is not an allowed character for the %1:s property."
-msgstr ""
+msgstr "\"%0:s\" ist kein erlaubtes Zeichen für die Eigenschaft %1:s"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: Missing dot.

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 12]
```diff
@@ -2607,94 +2602,88 @@ msgstr "&Passwort:"
...
 #: JvResources.pas:245
 msgid "Append"
-msgstr ""
+msgstr "Anhängen"
 
 #. Programmer's name for it: RsConnectionListImportAppendOverwriteExistingEntries
 #: JvResources.pas:246
 msgid "Append/Overwrite existing entries?"
-msgstr ""
+msgstr "Anhängen/überschreiben existierender Einträge?"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: "Überschreiben"

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 13]
```diff
@@ -2607,94 +2602,88 @@ msgstr "&Passwort:"
...
 
 #. Programmer's name for it: RsNoConnectionEntriesFound
 #: JvResources.pas:247
 msgid "No Connection Entries found!"
-msgstr ""
+msgstr "Keine Verbindungseinträge gefunden?"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: Why replace the ! with a ?

File: jvcl/locale/de/LC_MESSAGES/jvcl.po [hunk 14]
```diff
@@ -3030,6 +3018,8 @@ msgid ""
 "%0:s:\n"
 "%1:s"
 msgstr ""
+"%0:s:\n"
+"%1:s"
```
Comment (baka0815) [inline jvcl/locale/de/LC_MESSAGES/jvcl.po]: No translation needed if it's the same text.

**Comment by MHumm** [comment]
I fixed the things reported (some were already fixed by my further work) and will create a new pull request as soon as I have translated the remaining texts. The placeholder 1:1 translations have been done to have proper oversight over how far I got with this in the translation tool. So please do not mind this.

**Pull request merged**